values/time.go

```diff
@@ -149,47 +161,51 @@ func (d Duration) Equal(other Duration) bool {
 }
 
 // Duration will return the nanosecond equivalent
-// of this duration. It will assume that months are
-// the equivalent of 30 days.
+// of this duration. It will assume that all months are
+// the same length.
 //
 // It is recommended not to use this method unless
 // it is absolutely needed. This method will lose
 // any precision that is present in the Duration
 // and it should only be used for interfacing with
 // outside code that is not month-aware.
 func (d Duration) Duration() time.Duration {
```

Comment:
I wonder if this should have a scarier name, given that it could produce misleading results.

Reply:
Probably. I view this as likely a temporary method, but it always seems these things stick around for longer than "temporary". It was previously used though and I would prefer not to change the call site for a bunch of places when I don't want the function to exist anymore.